vue-components/src/components/HelloWorld.vue

```diff
@@ -38,17 +38,30 @@ export default Vue.extend( {
 
 <style lang="scss">
 .wikit-HelloWorld {
+	inset: logical 0 5px 10px;
+
 	&__headline {
-		margin: 40px 0 0;
+		margin-block-start: 40px;
+		margin-inline-end: 0;
+		margin-block-end: 0;
 	}
+
 	&__action-list {
 		list-style-type: none;
-		padding: 0;
+		padding-block-start: 0;
+		padding-inline: 0;
+		padding-block-end: 0;
+		padding-inline-end: 0;
```

Comment:
**Not really relevant - just for alignment**
```suggestion
```

As soon we as we have #117 in place we can omit this altogether as the opinionated reset will automatically set the 0 values for us.